benchmark/attri_util.py

```diff
@@ -170,12 +170,20 @@ class BenchmarkResult:
     result: List[BenchmarkMetrics]
 
     def __str__(self) -> str:
-        header = (
-            f"\nOperator: {self.op_name}  Performance Test (dtype={self.dtype}, mode={self.mode}, level={self.level})\n"
-            f"{'Size':<10} {'Torch Latency (ms)':>20} {'Gems Latency (ms)':>20} {'Gems Speedup':>20}"
-            f"{'Size Detail':>20}\n"
-            f"{'-' * 90}\n"
-        )
+        if self.op_name in ["mv","mm","addmm","bmm"]:
```

Comment:
This code is tailored to specific operators, so it would require updates each time we add new ones. Since BenchmarkResult is a general data class, I suggest improving the implementation to avoid relying on specific operator names here.

Reply:
ok